deploy/docker/docker-entrypoint.sh

```diff
@@ -110,13 +110,33 @@ CONFIG="${_EMQX_HOME}/etc/emqx.conf"
 CONFIG_PLUGINS="${_EMQX_HOME}/etc/plugins"
 for VAR in $(env)
 do
+    # echo value of $VAR_FULL_NAME
+    echo_value () { 
```

Comment:
It is recommended to put `echo_value` outside the for loop, and modify `echo_value` to the following content :
```
# echo value of $VAR_FULL_NAME
echo_value () {
    # check if contains sensitive value
    if [ ! -z $(echo $(echo $VAR_NAME | tr '.' ' ') |grep -w -o -E "password|passwd|key|token|secret") ]; then
        echo "***secret***"
    else
        eval echo \$$(echo $VAR_FULL_NAME)
    fi
}
```

This allows the `echo_value` method to match keywords more accurately, and reduces the number of loops

Reply:
resolved in 52c02a71dfc7f3331a1a80f5fcb52b9c42f410cc